models/app/features/panels/line-plot/reference.mdx

```diff
@@ -2,99 +2,145 @@
+    </Frame>
+- **Area plot**
+    <Frame>
+        <img src="/images/app_ui/plot_style_area_plot.png" alt="Area plot style"  />
+    </Frame>
+- **Percentage area plot:**
+    <Frame>
+        <img src="/images/app_ui/plot_style_percentage_plot.png" alt="Percentage plot style"  />
+    </Frame>
 
-`${run:displayName} - ${config:dropout}` will make the legend name for each run something like `royal-sweep - 0.5` where `royal-sweep` is the run name and `0.5` is the config parameter named `dropout`.
+## Grouping settings
+Aggregate all runs by turning on grouping, or group over an individual variable. When you turn on grouping in the runs table, the groups automatically populate into the graph.
 
-You can set value inside`[[ ]]` to display point specific values in the crosshair when hovering over a chart. For example `\[\[ $x: $y ($original) ]]` would display something like "2: 3 (2.9)"
+- **Group by**: Select a column. All runs with the same value in that column are grouped together.
+- **Aggregation**: The value of the line on the graph. Options are mean, median, min, and max of the group.
 
-Supported values inside `[[ ]]` are as follows:
+## Chart settings
+Configure titles and legend visibility:
 
-| Value         | Meaning                                    |
-| ------------  | ------------------------------------------ |
-| `${x}`        | X value                                    |
-| `${y}`        | Y value (Including smoothing adjustment)   |
-| `${original}` | Y value not including smoothing adjustment |
-| `${mean}`     | Mean of grouped runs                       |
-| `${stddev}`   | Standard Deviation of grouped runs         |
-| `${min}`      | Min of grouped runs                        |
-| `${max}`      | Max of grouped runs                        |
-| `${percent}`  | Percent of total (for stacked area charts) |
+- **Panel title**: Title displayed at the top of the panel.
+- **X-axis title**: Label for the x-axis.
+- **Y-axis title**: Label for the y-axis.
+- **Legend**: Show or hide the legend, and configure its position.
 
-## Grouping
+## Legend settings
 
-You can aggregate all of the runs by turning on grouping, or group over an individual variable. You can also turn on grouping by grouping inside the table and the groups will automatically populate into the graph.
+Customize the legend to show any config value logged and metadata from the runs, such as creation time or the user who created the run.
 
-## Smoothing
+### Legend template
 
-You can set the [smoothing coefficient](/models/support/formula_smoothing_algorithm/) to be between 0 and 1 where 0 is no smoothing and 1 is maximum smoothing.
+Define a template for the legend name.
 
+Example:
 
-## Ignore outliers
+`${run:displayName} - ${config:dropout}` produces a legend name like `royal-sweep - 0.5`, where `royal-sweep` is the run name and `0.5` is the config parameter named `dropout`.
```

Comment:
Add a sentence describing how they'd do this. As is, I have no idea where'd I add the above...